docs/geode-native-docs/transactions.html.md.erb

```diff
@@ -69,7 +69,7 @@ Both examples perform the same sequence of operations, displaying simple log ent
 - To run an example, follow the instructions in the `README.md` file in the example directory.
 - Review the source code in the example directory to see exactly how it operates.
 
-- You begin by running a script that sets up the servers-side environment by invoking `gfsh` commands to create a region, simply called "exampleRegion".
+- You begin by running a script that sets up the server-side environment by invoking `gfsh` commands to create a region, simply called "exampleRegion".
```

Comment:
Period/quotation marks again.

Also, just checking: Is this different from the README.md instructions/script mentioned above in this commit? The region in that case is called "example_userinfo."

Reply:
Different example with a different region name. Just taking this opportunity to fix a typo.